shared/lib_pv_io_manager.cpp

```diff
@@ -713,6 +713,7 @@ PVSystem_IO::PVSystem_IO(compute_module* cm, std::string cmName, Simulation_IO*
     dcNameplate = cm->as_double("system_capacity");
     //numberOfInvertersClipping = cm->as_integer("num_inverter_subhourly_clipping");
     numberOfInvertersClipping = dcNameplate / (Inverter->ratedACOutput / 1000);
+    if (numberOfInvertersClipping == 0.0) numberOfInvertersClipping += 1;
```

Comment:
Purely a semantics thing, but I think here you'd usually want to specify =1 rather than +=1, because it makes your intention clearer.